content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,160 @@ weight = 20
 
 +++
 
-## What is Kubeflow?
+This page describes how to install Kubeflow standalone components or Kubeflow Platform using package
+distributions or raw manifests.
 
-Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
-Operators can choose what is best for their users, there is no requirement to deploy every component.
+Read [the introduction guide](/docs/started/introduction) to
+understand what are Kubeflow standalone components and what is Kubeflow Platform.
 
-Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
-[Architecture](/docs/started/architecture/) pages.
+## Installing Kubeflow
 
-## How to install Kubeflow?
+You can install Kubeflow using one of these methods:
 
-Anywhere you are running Kubernetes, you should be able to run Kubeflow.
-There are two primary ways to install Kubeflow:
+- [**Install Kubeflow Components Standalone**](#install-kubeflow-components-standalone)
+- [**Install Kubeflow Platform from Packaged Distributions**](#install-kubeflow-platform-from-packaged-distributions)
+- [**Install Kubeflow Platform from Raw Manifests**](#install-kubeflow-platform-from-raw-manifests) <sup>(advanced users)</sup>
 
-1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
-1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
+### Install Kubeflow Components Standalone
 
-<a id="packaged-distributions"></a>
-<a id="install-a-packaged-kubeflow-distribution"></a>
+Some components in the [Kubeflow ecosystem](/docs/started/architecture/#conceptual-overview) may be
+deployed as standalone services, without the need to install the full platform. You might integrate
+these services as part of your existing AI/ML platform or use them independently.
 
-## Packaged Distributions of Kubeflow
+This is the easiest method to get started with Kubeflow ecosystem since those components usually
```

Comment:
perhaps change "easiest" to "a simple" or "an easier" method.    My point - is deploying a standalone component easier (for a user) than using vendor's SaaS offer ? maybe, maybe not.   Note - There might be a better word than simple .